src/test/java/org/folio/clients/InventoryClientTests.java

```diff
@@ -28,7 +29,8 @@
 import org.junit.jupiter.api.Test;
 
 public class InventoryClientTests {
-  private final WireMockServer fakeWebServer = new WireMockServer();
+  private final WireMockServer fakeWebServer = 
+      new WireMockServer(Options.DYNAMIC_PORT);
```

Comment:
The mod-patron change uses:
`
private final WireMockServer fakeWebServer = new WireMockServer(options().dynamicPort());
`

Why does this change use a different way of doing it (and split it across lines)?

Reply:
This file already had an options object in the imports that has similar functionality, so I used that.  It is split across lines because this module has an automated style-checker that prevents the module from compiling if your code doesn't conform to it's rules, and in this case, it considered the line too long.